base_location/README.rst

```diff
@@ -0,0 +1,115 @@
+====================================
+Location management (aka Better ZIP)
+====================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
+    :target: https://github.com/OCA/partner-contact/tree/14.0/base_location
+    :alt: OCA/partner-contact
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-base_location
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/134/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module introduces a zip model that allows you to manage locations in a better way.
+
+The zips will allow the users to complete automatically all address-related fields by just filling the zip.
```

Comment:
Actually not all address related fields. You will not be able to fill in house_number and door_number from the zip. In most countries also not the street.

Reply:
AFAIK, that fields are added through extension modules, so it's OK to say that with current dependency set.

Reply:
It is only a minor point anyway. Very glad with the module as is. But street is also not filled automatically, and this is definitely also address related.

Reply:
Yeah, I get your point about the street, although it's impossible to fill a not known data derived from the ZIP. If I read the text, I understand that the "address-related" text refers exactly to this: the address fields that are linked with the ZIP, not the not related one, and street (and the others) are not related, as a ZIP code contains a lot of streets.